pbspro.spec.in

```diff
@@ -513,6 +513,7 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
+%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```

Comment:
So, in master branch, there's a %exclude for pbs_schema_upgrade under pbs_execution as well. Do you think we should also add it there?

TBH, since this is kind of out of scope for your PR anyways, I would just leave these files as it is.

Reply:
ok, I agree
I have removed out of scope file changes,